source/00-config/mixins/_container-query.scss

```diff
@@ -0,0 +1,94 @@
+// Mixins: Container Query
+
+@use 'sass:string';
+@use '../config.settings' as *;
+@use '../functions' as *;
+
+// Create a min-width container query.
+// @param {Number} $breakpoint - width value.
+// @param {String} $container-name - name of container.
+@mixin container-query-min($breakpoint, $container-name: null) {
+  @if $container-queries-rems {
+    $breakpoint: rem($breakpoint);
+  }
+  @if $container-name {
+    // stylelint-disable
```

Comment:
I don't think we should be disabling the linter without comment (even though I see it's re-enabled a few lines later). I'm guessing this is because Stylelint doesn't understand something here? First recommendation would be to update Stylelint as part of this PR. Stylelint _should_ support `@container` now (their source code suggests they do), so Gesso might be running an old version. (Let me know if you need a hand with this.)

If even updating doesn't help Stylelint, we should [follow our own guidelines](https://github.com/forumone/gesso/blob/5.x/README.md#sass-linting) and include a comment about why the disabling of linting is necessary, so that we can (hopefully) remove that once Stylelint catches up.